cloudify_cli/config/parser_config.py

```diff
@@ -114,39 +311,36 @@ def parser_config():
                                 'dest': 'plugin_path',
                                 'type': argparse.FileType(),
                                 'required': True,
-                                'help': 'Path to the plugin file',
+                                'help': 'Path to a plugin file',
```

Comment:
well.. if you're already changing this, I'd make it clear that we're talking about a `Wagon` (there really isn't a term `plugin file`)